doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+{
+  "txid": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "hash": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "size": 109,
+  "vsize": 109,
+  "version": 1,
+  "locktime": 0,
+  "vin": [
+    {
+      "coinbase": "04d9eea3520101062f503253482f",
+      "sequence": 4294967295
+    }
+  ],
+  "vout": [
+    {
+      "value": 393004.00000000,
+      "n": 0,
+      "scriptPubKey": {
+        "asm": "0340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624 OP_CHECKSIG",
+        "hex": "210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac",
+        "reqSigs": 1,
+        "type": "pubkey",
+        "addresses": [
+          "D5TjotnkoXekNQBCn54CBWxjEzCJMMe7WS"
+        ]
+      }
+    }
+  ]
+}
+```
+
+`vin` and `vout` will give you information about transaction inputs and outputs.
+
+## Node configuration
+
+A server contain a lot of parameter to configure, from memory limits, to RPC credentials, ports... There is two ways to prepare your perfect node !
+
+Using directly `dogecoind` options, `--help` will display all configuration parameters.
+
+**Command example :**
+```bash
+$ dogecoind -testnet -datadir=/path/to/dir -rpcuser=Shibetoshi -rpcpassword=Nakamoto
+```
+
+Otherwise, a node is configurable through a `dogecoin.conf` file. Create it in the directory defined with the `datadir` setting, `$HOME/.dogecoin` by default, or specify the file location with `-conf`.
+
+**dogecoin.conf example :**
+```
+testnet=1
+datadir=/path/to/directory
+rpcuser=Shibetoshi
+rpcpassword=Nakamoto
+```
+You can see a more concrete example [here](/contrib/debian/examples/dogecoin.conf).
+
+### Mainnet, testnet and regnet
+
+To perform tests, for example to create a blockchain based application, you may need a sandbox environment. Multiple networks exists for this purpose.
+
+**Mainnet** : The main network where real transaction operate.  
+**Testnet** : The test network, with peers.  
+**Regnet** : The regression network, to test without peers and create blocks on-demand.
+
+*Mainnet* is the network used by default. To enable *testnet*, with `dogecoind` use the `-testnet` option and `testnet=1` as `dogecoin.conf` field.
+
+To enable *regnet*, use `-regtest` option.
+
+> **Tips :** Think to specify the network when you use `dogecoin-cli`.
+
+### RPC credentials
+
+A username and a password is needed to control the RPC interface. When no credentials are provided, dogecoin use a [special cookie](https://bitcoin.org/en/release/v0.12.0#rpc-random-cookie-rpc-authentication) to randomly generate it.
+
+For security reasons, because the RPC server may process sensitive data like private keys, you can define your own identifiers using `rpcuser` and `rpcpassword` settings.
```

Comment:
The cookie is generally more secure than the credentials though, because it's randomized on startup?

Instead, I'd recommend to UNDER NO CIRCUMSTANCE open the RPC port to the public internet.

Reply:
Changed by the following :
> A username and a password is needed to control the RPC interface. When no credentials are provided, dogecoin use a special cookie to randomly generate it. It's possible to define your own identifiers using rpcuser and rpcpassword settings.

I removed the misleading sentence about security.

Reply:
For now, sure, we catch grammar in round 2 ❤️